packages/vscode-common/getExtensionApi.ts

```diff
@@ -1,41 +1,42 @@
 import type {
   CommandServerApi,
+  ExcludableSnapshotField,
+  ExtraContext,
+  ExtraSnapshotField,
+  HatTokenMap,
   IDE,
   NormalizedIDE,
   SerializedMarks,
+  SnippetMap,
   TargetPlainObject,
+  TestCaseSnapshot,
   TextEditor,
 } from "@cursorless/common";
-import type {
-  ExcludableSnapshotField,
-  ExtraContext,
-  ExtraSnapshotField,
-  Graph,
-  TestCaseSnapshot,
-  ThatMark,
-} from "@cursorless/cursorless-engine";
 import * as vscode from "vscode";
 import type { SyntaxNode } from "web-tree-sitter";
-import type { SnippetMap } from "../cursorless-engine/snippets/snippet.types";
-import type { Target } from "../cursorless-engine/typings/target.types";
 
-interface TestHelpers {
-  graph: Graph;
+export interface TestHelpers {
   ide: NormalizedIDE;
   injectIde: (ide: IDE) => void;
 
-  // FIXME: Remove this once we have a better way to get this function
-  // accessible from our tests
-  plainObjectToTarget(
+  hatTokenMap: HatTokenMap;
```

Comment:
Note that the `HatTokenMap` is now [an interface](https://github.com/cursorless-dev/cursorless/pull/1280/files#diff-1bef5c2e48c3c43737fa37e4ccbbfe57fa9354e889dd3880d05a13ef845cd79b), whereas before it was the type of [the class](https://github.com/cursorless-dev/cursorless/pull/1280/files#diff-9f94f61c925ef52b2b18b010d6ba7d6a5d8ac7367bf989b55ef242ff47cb01f5) that implemented our hat token map